rmgpy/solver/simple.pyx

```diff
@@ -163,6 +169,15 @@ cdef class SimpleReactor(ReactionSystem):
                     conditions[species_dict[label]] = value
         self.sens_conditions = conditions
 
+    def get_const_spc_indices (self, coreSpecies):
+        "Allow to identify constant Species position in solver"
+        for spc in self.const_spc_names:
+            if self.const_spc_indices is None: #initialize once the list if constant SPC declared
```

Comment:
Sorry, this is quite picky, but comments should have two spaces before the `#` and one space after:
```
if self.const_spc_indices is None:  # initialize
```